### PR TITLE
Allow admins to migrate interactives

### PR DESCRIPTION
**Pull request by emcelroy**
PT Story: https://www.pivotaltracker.com/story/show/178060576

[#178060576]

These changes provide an option to migrate library interactives used by questions in existing activities from one version to another.

### CODE REVIEW

File: spec/controllers/library_interactives_controller_spec.rb [hunk 1]
```diff
@@ -111,6 +111,23 @@
...
+      end
+    end
+
+    describe "PUT #migrate" do
+      it "changes all references to one library_interactive to another specified library_interactive and redirects to the library_interactive index" do
+        library_interactive1 = LibraryInteractive.create! valid_attributes
+        library_interactive2 = LibraryInteractive.create! valid_attributes
+        put :update, {:id => library_interactive1.to_param, :new_library_interactive_id => library_interactive2.to_param}
+        expect(response).to redirect_to(library_interactives_url)
+      end
```
Comment (scytacki) [inline spec/controllers/library_interactives_controller_spec.rb:128]: I think this test is not actually covering the migration function. Code climate is reporting it as not covered. My guess is that the `put :update` needs to be changed to `put :migrate`

**Review by scytacki — CHANGES_REQUESTED** [review]
I think one of the tests is covering the wrong action.

File: spec/controllers/library_interactives_controller_spec.rb (hunk 1 above)
Comment (emcelroy) [reply spec/controllers/library_interactives_controller_spec.rb]: Right. Thanks for catching that. It's fixed.

**Review by scytacki — APPROVED** [review]
Looks great!

**Pull request merged**